### PR TITLE
Add get tables method

### PR DESCRIPTION
**Pull request by ondrajodas**
Solving part of - https://keboola.atlassian.net/browse/COM-24

### CODE REVIEW

File: src/Keboola/DbExtractor/Extractor/Firebird.php [hunk 1]
```diff
@@ -50,7 +51,66 @@ public function testConnection(): void
...
+				END AS NULLABLE
+			FROM RDB\$RELATION_FIELDS r
+				LEFT JOIN RDB\$FIELDS f ON r.RDB\$FIELD_SOURCE = f.RDB\$FIELD_NAME
+			WHERE r.RDB\$RELATION_NAME = '$table'
+			ORDER BY r.RDB\$FIELD_POSITION;"
+        );
+        $columns = [];
+        foreach ($resultColumns as $column) {
+            $key = $column['FIELD_NAME'];
+            $baseType = new GenericStorage(trim($column['FIELD_TYPE']));
```
Comment (pivnicek) [inline src/Keboola/DbExtractor/Extractor/Firebird.php]: I think it's great, but I'm afraid that there may be a couple of missing types in the `GenericStorage` class that could result in some false matching.  For example I don't think `GenericStorage` contains `D_FLOAT` or `QUAD` . (I actually don't know what quad is:).

We can either change for example `D_FLOAT` to `FLOAT` here, or add them to the `GenericStorage` class.  To necham na tobe.

**Review by pivnicek — CHANGES_REQUESTED** [review]
Just one comment about the data types

File: src/Keboola/DbExtractor/Extractor/Firebird.php (hunk 1 above)
Comment (ondrajodas) [reply src/Keboola/DbExtractor/Extractor/Firebird.php]: @pivnicek ok so i add PR in php-datatypes - https://github.com/keboola/php-datatypes/pull/25

`quad` is `Quadruple precision floating point format` => float :)

**Review by pivnicek — APPROVED** [review]
So as long as you update the lib here cajk 👍

**Comment by ondrajodas** [comment]
updated

**Pull request merged**